jsapp/scss/components/_kobo.modal.scss

```diff
@@ -348,6 +348,81 @@ $z-modal-x: 10;
   }
 }
 
+.modal.modal-media-preview {
+  border-radius: 6px;
+  min-width: 25%;
+
+  .modal__content {
+    background-color: $kobo-gray-98;
+  }
+
+  .modal__header {
+    min-height: 0;
+    padding: 16px;
+    background-color: $kobo-white;
+    color: $kobo-gray-24;
+
+    .modal__x {
+      padding: 0;
+      margin: 0;
+    }
+
+    .table-media-preview-header {
+      width: 100%;
+      display: flex;
+      justify-content: space-between;
+      font-size: 14px;
+      font-weight: 600;
+      margin-right: 12px;
+
+      .table-media-preview-header__title {
+        padding-top: 3px;
+
+        .table-media-preview-header__label {
+          vertical-align: top;
+        }
+
+        i.k-icon {
+          color: $kobo-gray-65;
+          margin-right: 6px;
+        }
+      }
+
+      .kobo-light-button {
+        padding-right: 0px;
+        i.k-icon-download {
+          font-weight: 800;
+          margin-left: 3px;
+        }
+      }
+    }
+  }
+
+  .modal__body {
+    padding: 0;
+    margin: 0;
+
+    .table-media-preview {
+      padding: 32px 0 12px 0;
+      text-align: center;
+      max-width: 600px;
+
+      .audio-player {
+        margin-right: 12px;
+      }
+
+      img, video {
+        width: 100%;
```

Comment:
Do you think it would be more sensible for the image and the video to scale to fit in smaller (small height) screens (e.g. smartphone horizontally)?
<img width="623" alt="Screenshot 2021-07-21 at 18 00 39" src="https://user-images.githubusercontent.com/2521888/126529545-2b199113-5874-437d-a6c4-b17f46f197b2.png">

Reply:
Weird it's pretty responsive to me, can you send the video you used to take this?

Reply:
Here it is: https://user-images.githubusercontent.com/2521888/127569757-dac2e0aa-fa46-42d4-8b1e-04a23fff3c07.mov

But it's actually a minor thing, as most probably you'd watch the video in fullscreen on small devices

Reply:
What I thought would be a better solution is to scale the `<video>` in such case, so it would look like this:
<img width="613" alt="Screenshot 2021-07-29 at 22 52 26" src="https://user-images.githubusercontent.com/2521888/127570621-8c5d1779-eeb2-4a79-9657-e74efc70171d.png">

Reply:
I'm not entirely sure how to approach it though, so I would rather note it down somewhere as an improvements idea :)

Reply:
Made an issue #3392 :D